src/pykeen/utils.py

```diff
@@ -799,15 +800,36 @@ def strip_dim(*tensors: torch.FloatTensor, n: int = 4) -> Sequence[torch.FloatTe
 def upgrade_to_sequence(x: Union[X, Sequence[X]]) -> Sequence[X]:
     """Ensure that the input is a sequence.
 
+    .. note ::
+        While strings are technically also a sequence, i.e.,
+
+        .. code-block:: python
+            isinstance("test", Sequence) is True
+
+        this may lead to unexpected behaviour when calling `upgrade_to_sequence("test")`.
+        We thus handle strings as non-sequences. To recover the other behaviour, the following may be used:
+
+        .. code-block:: python
```

Comment:
I don't think sphinx accepts code block types anymore

Reply:
So we should drop this, i.e.
```suggestion
        .. code-block::
```
? will this interfere with `doctest`?

Reply:
yes, drop that